tst/regression/test_suites/restart/restart.py

```diff
@@ -38,7 +38,8 @@ def Prepare(self, parameters, step):
             parameters.driver_cmd_line_args = [
                 '-r',
                 'gold.out0.00001.rhdf',
-                'parthenon/job/problem_id=silver'
+                'parthenon/job/problem_id=silver',
+                'parthenon/mesh/nx1=%d' % (64 * parameters.num_ranks)
```

Comment:
One issue I see with setting the number of cells equal to the number of ranks is that for tests that compare to a gold file, the test will fail when run with a different number of cores as the gold file.

We should document somewhere how many cores the gold files were run with so we can reproduce on non-CI machines.

Reply:
Hmm... this is interesting I have been running the regression tests and have not noticed them failing.

Reply:
Huh. Maybe I'm misunderstanding how the gold comparisons work... but I have trouble understanding how that can pass. If the gold file has a different number of cells than the test output file, you *should* get a failure.

Reply:
If we're not getting a failure in that case, I'd argue the regression testing infrastructure has a bug.

Reply:
Well let me check before we investigate, I might not have run them all the way through.

Reply:
Yes, this was failing. I guess we will need a different gold standard file if we want to test against more than one gpu.